src/include/sof/lib/perf_cnt.h

```diff
@@ -82,6 +82,37 @@ struct perf_cnt_data {
 		}							  \
 	} while (0)
 
+/**
+ * For simple performance measurement and optimization in development stage,
+ * tik-tok api is provided. Performance data are traced at each tok call,
```

Comment:
Is this a time measurement API ? If so it's probably better to rename the API calls

Reply:
There is a gentle difference - in already used `perf_cnt_stamp` trace method is called when new cpu peak will be reached, what is good when we are debugging whole system (much less trace messages). In tik-tok each toc call is related with data print, so it's designed for single component / function optimization, otherwise number of trace data will be overwhelming.

Reply:
ok, but we need to call it the same name, it's called tic toc and tik tok in different places.

Reply:
sure, it was a mistake - already corrected